csrankings-j.csv

```diff
@@ -567,7 +567,7 @@ Jayanta Mukhopadhyay,IIT Kharagpur,http://www.facweb.iitkgp.ac.in/~jay,NOSCHOLAR
 Jayanthi Sivaswamy,IIIT Hyderabad,https://www.iiit.ac.in/people/faculty/jsivaswamy,HfATfbMAAAAJ
 Jayme Luiz Szwarcfiter,UFRJ,http://www.cos.ufrj.br/~jayme,cjQH4G0AAAAJ
 Jayne Wu,University of Tennessee,http://www.eecs.utk.edu/people/faculty/jwu10,oXoWM_QAAAAJ
-Jayson Boubin,Binghamton University,https://www.binghamton.edu/computer-science/people/profile.html?id=jboubin,JwIpWcwAAAAJ
+Jayson G. Boubin,Binghamton University,https://www.binghamton.edu/computer-science/people/profile.html?id=jboubin,JwIpWcwAAAAJ

```

Comment:
Looks like we need both entries - see DBLP.

Reply:
I just added the second entry back